tox.ini

```diff
@@ -50,10 +50,10 @@ description = Apply coding style standards to code
 commands =
     # uncomment the following line if this charm owns a lib
     # codespell {[vars]lib_path}
-    codespell {toxinidir}/. --skip {toxinidir}/.git --skip {toxinidir}/.tox \
-      --skip {toxinidir}/build --skip {toxinidir}/lib --skip {toxinidir}/venv \
-      --skip {toxinidir}/.mypy_cache \
-      --skip {toxinidir}/icon.svg --skip *.json.tmpl
+    codespell {toxinidir}/. --skip {toxinidir}/./.git --skip {toxinidir}/./.tox \
```

Comment:
Can you explain what issue you are having without this change?

Reply:
Those directories are not ignored by `codespell` if `/./` is not there. This is pretty much how it is done in other repos as well.

Reply:
I suggest splitting this and the library bump up into a separate PR so it is clear what fixes #78 and what is maintenance

Reply:
+1 on the above suggestion, please split this PR with those maintenance/CI fixes.

Reply:
Got it. Will do.